Apps/AdminWebClient/src/Server/Controllers/AuthenticationController.cs

```diff
@@ -58,9 +60,10 @@ public AuthenticationController(IAuthenticationService authenticationService, IL
         /// <returns>The authentication model representing the current ASP.Net Core Authentication cookie.</returns>
         [HttpGet]
         [ProducesResponseType(200)]
-        public AuthenticationData GetAuthenticationData()
+        public async Task<AuthenticationData> GetAuthenticationData()
```

Comment:
instead of making async you can wrap the async call below in a Task.Run()

Reply:
I moved this back into the auth service and wrapped it in Task.Run